internal/env/env.go

```diff
@@ -2,6 +2,10 @@ package env
 
 import "os"
 
+func IsDocsRuntime() bool {
```

Comment:
Nit: It'd be really helpful for the future if this was documented with what this is and why its here

```suggestion
// Returns true if the CLI_RUNTIME environment variable is set to "docs".
// This environment variable is used to determine when website documentation
// is being rendered to prevent unexpected CLI formatting characters.
func IsDocsRuntime() bool {
```